src/libs/actions/Transaction.js

```diff
@@ -87,6 +93,12 @@ function saveWaypoint(transactionID, index, waypoint) {
             },
         },
     });
+    const recentWaypointAlreadyExists = _.find(recentWaypoints, (recentWaypoint) => recentWaypoint.address === waypoint.address);
```

Comment:
FYI this caused a [deploy blocker](https://github.com/Expensify/App/issues/26286)